msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/CIAMAuthority.java

```diff
@@ -6,7 +6,7 @@
 import java.net.MalformedURLException;
 import java.net.URL;
 
-public class CIAMAuthority extends Authority{
+public class CIAMAuthority extends Authority {
```

Comment:
Pls add a unit test to ensure that OIDC discovery doesn't happen twice for a Generic Authority.